apps/docs/pages/guides/cli/local-development.mdx

```diff
@@ -147,95 +177,195 @@ Database changes are managed through "migrations." Database migrations are a com
   ></iframe>
 </div>
 
-### Make database changes
+For this guide, we'll create a table called `employees` and see how we can make changes to it.
 
-For this guide, create a table called `employees`. In Supabase Studio, navigate to the **SQL Editor** page and run the following SQL command:
+<StepHikeCompact>
 
-```sql
-create table
-  employees (
-    id integer primary key generated always as identity,
-    name text
-  );
+  <StepHikeCompact.Step step={1}>
+    <StepHikeCompact.Details title="Create your first migration file">
+
+      To get started, generate a [new migration](https://supabase.com/docs/reference/cli/supabase-migration-new) to store the SQL needed to create our `employees` table
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+
+```bash
+supabase migration new create_employees_table
 ```
 
-<Admonition type="note">
+    </StepHikeCompact.Code>
 
-You can execute any SQL using the `DB URL` shown by [`supabase status`](/docs/reference/cli/usage#supabase-status).
+  </StepHikeCompact.Step>
+</StepHikeCompact>
 
-</Admonition>
+<StepHikeCompact>
 
-Run the [`db diff`](/docs/reference/cli/usage#supabase-db-diff) command to detect changes in the local database:
+  <StepHikeCompact.Step step={2}>
+    <StepHikeCompact.Details title="Add the SQL to your migration file">
+      This creates a new migration: supabase/migrations/\<timestamp\>
+        _create_employees_table.sql.
 
-```sh
-supabase db diff create_employees -f create_employees
-```
+        To that file, add the SQL to create this `employees` table
+    </StepHikeCompact.Details>
 
-This creates a new migration named `supabase/migrations/<timestamp>_create_employees.sql`, representing any changes made to the local database since [`supabase start`](/docs/reference/cli/usage#supabase-start).
+    <StepHikeCompact.Code>
+      ```sql
 
-### Add sample data
+create table
```

Comment:
I think we should think about this sql statement as its own "how-to" or reference. 

A user should ideally be able to create a new table from this example really easily. What other data types should we include in here that would be useful for people? 

Should the id primary key be an integer or something else? A uuid?

Reply:
suggestions here: 
an auto-incrementing id (serial?) 
a number
a date